pandas/tests/indexing/multiindex/test_multiindex.py

```diff
@@ -83,3 +83,32 @@ def test_nested_tuples_duplicates(self):
         df3 = df.copy(deep=True)
         df3.loc[[(dti[0], "a")], "c2"] = 1.0
         tm.assert_frame_equal(df3, expected)
+
+
+def test_combine_first_with_nan_multiindex():
```

Comment:
move to pandas/tests/frame/methods/test_combine_first.py

Reply:
done